composer.json

```diff
@@ -10,17 +10,15 @@
     "homepage": "https://github.com/laravel-enso/charts",
     "type": "library",
     "license": "MIT",
-    "authors": [
-        {
-            "name": "Adrian Ocneanu",
-            "email": "aocneanu@gmail.com",
-            "homepage": "https://laravel-enso.com",
-            "role": "Developer"
-        }
-    ],
+    "authors": [{
+        "name": "Adrian Ocneanu",
+        "email": "aocneanu@gmail.com",
+        "homepage": "https://laravel-enso.com",
+        "role": "Developer"
+    }],
     "require": {
         "laravel/framework": "^7.0",
-        "laravel-enso/helpers": "1.15.*",
+        "laravel-enso/helpers": "^2.0",
```

Comment:
are we missing the enum dependency?

Reply:
we didn't change enums package in this package.

Reply:
ah right, it's a local enum